docs/environment-variables.md

```diff
@@ -0,0 +1,36 @@
+# Environment Variables
+
+This document outlines the set of environment variables that can be used to customize the behaviours at different levels.
```

Comment:
I think I'm wrong about the text for this. I think we should not emphasize they are unsupported, more that they are typically added to test out experimental features and should not be needed by most users.

Reply:
That sounds much better actually. I just pasted the exact sentence you said but feel free to modify further.

Reply:
I would actually explicitly say that these may be removed at any time